test/metabase/sync/sync_metadata/comments_test.clj

```diff
@@ -114,7 +117,14 @@
       (mt/dataset (basic-table "table_with_comment_after_sync" nil)
         ;; modify the source DB to add the comment and resync
         (driver/notify-database-updated driver/*driver* (mt/db))
-        (tx/create-db! driver/*driver* (basic-table "table_with_comment_after_sync" "added comment"))
+        (tx/create-db! driver/*driver* (basic-table "table_with_comment_after_sync" nil))
```

Comment:
This test relied on the fact that descriptions are not included when inserting tables, the correct behavior is to start with a table without a description, then sync, then add a description, then sync. see https://github.com/metabase/metabase/pull/36677/files#r1427842665 for context.